source/_docs/lockr.md

```diff
@@ -0,0 +1,162 @@
+
+
+### WP-CLI Commands
+
+The Lockr plugin contains a number of WP-CLI commands to make registering the site, and setting/getting a key simple through the command line.
+
+```nohighlight
+lockr register-site --email=[<Lockr email address>] --password=[<Lockr account password>]
+```
+This command will register the site with Lockr to the email address provided. The password is only necessary for email addresses already with a Lockr account. This is useful for automated deployment from a custom upstream using Quicksilver.
+
+
+```nohighlight
+lockr lockdown
+```
+Run this command and Lockr will go to a [patch library](https://github.com/lockr/lockr-patches/tree/wp) and automatically patch your existing plugins that do not currently integrate natively with Lockr.
+
+
+```nohighlight
+lockr get-key [key name]
+```
+This command will get the key from Lockr and decrypt it. This is a useful command to program in automated functionality in Quicksilver
+
+
+```nohighlight
+lockr set-key --name=[key name] --label=[key label] --value=[key value]
+```
+This command will encrypt a key and send it to Lockr. This is useful during site migrations or automated deployments of new sites through Quicksilver.
+
+
+
+## Drupal Installation
+
+Lockr provides a simple-to-use [plugin](https://www.drupal.org/project/lockr) for Drupal’s [Key Module](https://www.drupal.org/project/key) to manage your site’s keys and secrets. By pairing together Lockr, [Encrypt](https://www.drupal.org/project/encrypt) and the Key module Drupal can finally take advantage of best security practices of separating the keys away from the data they protect. Lockr is currently available for Drupal 7 and Drupal 8. Visit our [Github page](https://github.com/lockr/lockr-patches/tree/drupal7) for a list of plugins that can be automatically patched.
+
+### Installing through the administration page
+
+1. Lockr has a dependency on the Key module download the corresponding versions for your installation of Drupal and place them in the modules folder (Drupal 7: `/sites/all/modules/contrib` Drupal 8: `/modules/contrib`). 
+2. Once uploaded to the server, go to the modules page to enable the modules, and their dependencies (ex. `http://my-site.pantheonsite.io/admin/modules`). 
+3. After the modules are successfully installed, go to the Lockr configuration page to register your site (ex. `http://my-site.pantheonsite.io/admin/config/system/lockr`).
+
+![Drupal Lockr Configuration example](/source/docs/assets/images/lockr_drupalconfig.png)​
+
+4. Enter your email address and Lockr will take care of the rest! (note: For first time users, you will be receiving an email with instruction, if an existing Lockr users should use the email address they used to create their Lockr account. After submitting the request to register the site, you will be prompted for your Lockr password to confirm adding the site to your account.)
+
```

Comment:
1. [Set the connection mode to SFTP](/docs/sftp/) for the Dev or Multidev environment via the Pantheon Dashboard or with [Terminus](/docs/terminus):
 
 ```
 terminus site set-connection-mode --mode=sftp
 ```

2. Install the [Lockr](https://www.drupal.org/project/lockr) module and the required [Key](https://www.drupal.org/project/key) module by uploading files to the modules directory (Drupal 7: `/sites/all/modules/contrib` Drupal 8: `/modules/contrib`) or using the [Drupal interface](https://www.drupal.org/docs/7/extending-drupal/installing-modules). 
  
3. Once uploaded to the server, navigate to the modules page (`/admin/modules`) and enable both modules.

4. Once enabled, go to the Lockr configuration page (`/admin/config/system/lockr`):
 
  ![Drupal Lockr Configuration example](/source/docs/assets/images/lockr_drupalconfig.png)​
 
5.  Enter your email address and click **Sign Up**.  
 
 <div class="alert alert-info">
 <h4>Note</h4>
 First time Lockr users will receive an email with account information. Existing Lockr users should provide the email address associated with your Lockr account and authenticate when prompted. 
 </div>